test/performance/latency/xml/latency_interprocess_best_effort_tcp_profile.xml

```diff
@@ -38,12 +38,7 @@
                 <name>latency_test_publisher</name>
             </rtps>
         </participant>
-        <publisher profile_name="pub_publisher_profile">
-            <topic>
-                <name>latency_interprocess_pub2sub</name>
-                <dataType>LatencyType</dataType>
-                <kind>NO_KEY</kind>
-            </topic>
+        <data_writer profile_name="pub_publisher_profile">
```

Comment:
Because those topic elements are no longer supported by the schema, and the latency test does not need those elements for its effectiveness. If I understood well, that topic xml configuration was not beeing implemented by the parser, so that "noise" code was removed from the specifications (schema and documentation). @EduPonz was that the reason?